src/main/java/de/retest/web/WebElementPeer.java

```diff
@@ -126,10 +126,7 @@ private boolean isDefault( final String attribute, final String attributeValue )
 		if ( attributeValue.equals( "normal" ) ) {
 			return true;
 		}
-		if ( AttributesProvider.getInstance().getSizes().contains( attribute ) && attributeValue.equals( "0px" ) ) {
-			return true;
-		}
-		return false;
+		return AttributesProvider.getInstance().getSizes().contains( attribute ) && attributeValue.equals( "0px" );
```

Comment:
I wouldn't do this. As you already said, this is not an improvement but a false positive from SonarCloud. Would you deleting the commit? I can show you how to fix this in SonarCloud.